### ISSUE TITLE
Peer Review - Midterm Report - jc2498

### ISSUE THREAD

**Issue by Yazmine1213C** [YyRrHao/ORIE_5741_GrpProj#8]
What's it about? 
Make prediction on Bitcoin price.
What data are they using? 
Using Yahoo finance package of bitcoin, gold, crude oil and also combine the popularity of tweets which is getting from Twitter API.
What's their objective? 
By using the bitcoin price, the comparison with gold and crude oil, and the comment of twitters related to bitcoin to understand its price trend, and make the forecast by using the ARIMA model.	
Three things I like:
The topic is really attractive, stock prediction is always an enticing prospect especially the cryptocurrency is the hottest subject that everyone talks about it.
The whole structure of the paper is neat and organized, the data visualizations are in detail.
Using various techniques and two different models to make the data relationship prediction interpretable.  
Three things to improve:
The EDA is perfect and understandable, but for the model part, maybe gives more explanations including model formula, loss function choosing, feature importance.
For the VAR model, not fully understand what is this and how it works. So in your impulse response analysis, from figure how you know the relationship of gold price and oil price.
As you mentioned at the end, totally agreed that to have more related features and more samples to have a more generalized model. While the overfitting issue also should be kept in mind when the model complexity is increasing.
